arm_robots/src/arm_robots/cartesian.py

```diff
@@ -22,22 +21,18 @@ def pose_distance(a: Pose, b: Pose, rot_weight=0):
     return pos_distance + rot_distance
 
 
-class ArmSide(enum.IntEnum):
-    LEFT = 0
-    RIGHT = 1
-
-
 class CartesianImpedanceController:
     def __init__(self, tf_buffer, motion_status_listeners, motion_command_publisher, joint_lim_low, joint_lim_high,
+                 world_frame_name, sensor_frame_names=None,
                  position_close_enough=0.0025, timeout_per_m=500, intermediate_acceptance_factor=7.,
                  joint_limit_boundary=0.03):
         """
 
         :param tf_buffer: tf2 Buffer object
         :param motion_status_listeners: ROS listener (wrapper around subscriber) for status messages
         :param motion_command_publisher: ROS publisher for arm commands
-        :param joint_lim_low: lower joint limits in degrees
-        :param joint_lim_high: upper joint limits in degrees
+        :param joint_lim_low: lower joint limits in degrees or radians
```

Comment:
this seems really sketchy. I'm not sure why you want to support both and try to check which was passed but it seems like a bad idea...

Reply:
Ok, instead of implicitly converting to radians based on heuristics, now it'll just warn the user if the heuristics activate indicating that it may not be valid radians